.github/workflows/style.yml

```
@@ -23,4 +23,9 @@ jobs:
           pip install -r .github/workflows/requirements/style.txt
       # Update output format to enable automatic inline annotations.
       - name: Run Ruff
-        run: ruff check --output-format=github src
+        run: |
+          ruff check --output-format=github
+          ruff format --check --diff
```

could we add this to make sure that imports get sorted? there doesn't seem to be a unified command yet. see https://docs.astral.sh/ruff/formatter/#sorting-imports

```suggestion
          ruff check --select I --fix
          ruff format --check --diff
```